detekt-rules/build.gradle.kts

```diff
@@ -17,6 +17,10 @@
  * License-Filename: LICENSE
  */
 
+import org.gradle.internal.logging.text.StyledTextOutput
```

Comment:
Commit message: Are the double brackets intentional?

Reply:
Yes, as that way it renders like:

![image](https://user-images.githubusercontent.com/349154/134550569-e3904704-5d81-4b8a-9124-0fec81689aaf.png)

Reply:
I see, so you want the brackets to show in the PR description. I would prefer to stick with the formatting we have used so far, for consistency, and because double brackets only look good if the commit message is directly used for the PR description for single commit PRs, but they look bad in all other places.

Reply:
I've removed the double square brackets.

Reply:
Thanks.